ci_framework/roles/libvirt_manager/templates/ci-network-data.yml.j2

```diff
@@ -0,0 +1,50 @@
+{#
+The ci_job_networking is exposed by the reproducer role. It namely exposes
+the CI job configuration passed down to zuul so that we can ensure we're
+using the same networks.
+#}
+{% set ns = namespace(
+    ip4='',
+    gw4='',
+    iface=cifmw_reproducer_private_nic,
+    config_nm=false, ip_suffix=0)
+-%}
+{% if vm_type == 'controller' -%}
```

Comment:
Reading this config is very difficult. Some values are duplicated, other are overriden.
Can you rewrite it into `if`/`elif`/`else` containing full set of values?
Something like:
```
{% if vm_type == 'controller' -%}
{% set ns.ip4 = ... %}
{% set ns.gw4 = ... %}
{% set ns.iface = cifmw_reproducer %}
{% set ns.config_nm = false %}
{% set ns.ip_suffix = 0 %}
{% elif == ... %}
{% set ns.ip4 = ... %}
{% set ns.gw4 = ... %}
{% set ns.iface = cifmw_reproducer %}
{% set ns.config_nm = false %}
{% set ns.ip_suffix = 0 %}
{% else %}
{% set ns.ip4 = ... %}
{% set ns.gw4 = ... %}
{% set ns.iface = cifmw_reproducer %}
{% set ns.config_nm = false %}
{% set ns.ip_suffix = 0 %}
{% endif %}
```

Or _at least_ keep order of values the same for all conditions?
It's very hard to track all variables between different places.

Reply:
some values don't need to be overridden - so I won't repeat them. Re-ordering, done.